docs/wiki/boards/current/SAKURAH743/SAKURAH743.mdx

```diff
@@ -237,11 +237,11 @@ import SpecGrid from '@site/src/components/SpecGrid'
 <TabItem value="photos" label="Photos">
 ![TOP](./SAKURAH743-images/SkySakuraH743_top.png)
 ![BOTTOM](./SAKURAH743-images/SkySakuraH743_bottom.png)
-![TOPpit](./SAKURAH743-images/SkySakuraH743_TOP.jpg)
-![BOTTOMpitwithpower](./SAKURAH743-images/SkySakuraH743_bot_P.jpg)
-![BOTTOMpit](./SAKURAH743-images/SkySakuraH743_bot.jpg)
-![POWERtop](./SAKURAH743-images/SkySakuraH743_powerB.jpg)
-![POWERbot](./SAKURAH743-images/SkySakuraH743_powerT.jpg)
+![TOPpit](./SAKURAH743-images/SkySakuraH743_TOP.JPG)
```

Comment:
:thinking: File extensions should be lowercase, rather change the source.

Reply:
Changing the image filetype to lowercase wasn't detected as any change, it wouldn't let me commit an empty commit. Unsure what's going on there